src/components/ConnectCableDialog.tsx

```diff
@@ -28,7 +29,14 @@ const configs: Record<ConnectionFlowType, Config> = {
   [ConnectionFlowType.RadioRemote]: {
     headingId: "connectMB.connectCableMB1.heading",
     subtitleId: "connectMB.connectCableMB1.subtitle",
-    onLink: LinkType.None,
+    ...(stage === "local"
+      ? {
+          linkTextId: "connectMB.connectCable.skip",
+          onLink: LinkType.Skip,
+        }
+      : {
+          onLink: LinkType.None,
+        }),
```

Comment:
For radio connection dev purposes, when locally developing, a skip button is visible for micro:bit 1 and the device Id is stored in the local storage.